apps/ledger-live-mobile/src/components/FabActions/index.tsx

```diff
@@ -84,6 +84,7 @@ export const FabButtonBarProvider = ({
   eventProperties?: { [key: string]: unknown };
   children: (value: { quickActions: ActionButtonProps[] }) => ReactNode;
 }) => {
+  const { track } = useAnalytics();
```

Comment:
Whiat is different between?
```
import { track } from "../../analytics";
``` 
and 
```
const { track } = useAnalytics()
```

Reply:
Good question. I used `useAnalytics` as it calls `track` and adds in the current `page` route automatically. However, when I've been testing this, `page` is sometimes wrongly assigned to `portfolio_navigator`.  I also have just noticed that `track`  also adds in a `screen` value which does look correct.

 It's a little bit confusing. I think we should send either `page` or `screen` and ensure that this is set to the right value, but don't really want to fix this issue in this PR as it'll affect all analytics.